### ISSUE TITLE
EngineError / failure to parse after upgrade to 0.2.0-193

### ISSUE THREAD

**Issue by rntz** [talonvoice/talon#435]
I haven't managed to minimize this yet, but I'm getting a EngineError if I say 'terminal' under 193 that I wasn't under 171:

```
2021-11-26 15:17:45 ERROR cb error topic="phrase" cb=<bound method SpeechSystem.engine_event of <talon.scripting.speech_system.SpeechSystem object at 0x7fe447e2b590>>
   19:       lib/python3.9/threading.py:930* # cron thread
   18:       lib/python3.9/threading.py:973* 
   17:       lib/python3.9/threading.py:910* 
   16:                    talon/cron.py:155| 
   15:                    talon/cron.py:106| 
   14:                     talon/vad.py:21 | 
   13: talon/scripting/speech_system.py:355| 
   12:             talon/engines/w2l.py:711| 
   11:      talon/scripting/dispatch.py:105| 
   10:      talon/scripting/dispatch.py:144| 
    9:      talon/scripting/dispatch.py:133| 
    8:          talon/scripting/rctx.py:233| # 'phrase' main:_redispatch()
    7: talon/scripting/speech_system.py:62 | 
    6:      talon/scripting/dispatch.py:105| 
    5:      talon/scripting/dispatch.py:144| 
    4:      talon/scripting/dispatch.py:133| 
    3: ------------------------------------# cron thread
    2:          talon/scripting/rctx.py:233| # 'phrase' main:engine_event()
    1: talon/scripting/speech_system.py:399| 
talon.engines.EngineError: failed to parse phrase: ['terminal']
```

I don't think I have the bare command 'terminal' mapped anywhere, and when I say 'terminal' on 171 it gets rejected. When I sim('terminal') on either 171 or 193 it fails to parse, as expected. I don't have time to try to minimize this right now unfortunately.

**Comment by lunixbochs** [comment]
this was an issue if a beam ended on the accepting state of a list, but the grammar stack itself was not in an accepting state. will be fixed in next beta.

**Comment by lunixbochs** [comment]
fixed in 0.2.0-196
